### PR TITLE
Add missing control+shift combination for web

### PR DESCRIPTION
**Pull request by azimgd**

### CODE REVIEW

**Comment by luacmartins** [comment]
@azimgd can you merge main?

**Comment by azimgd** [comment]
moment.

**Comment by azimgd** [comment]
done.

File: ios/KeyCommand.mm [hunk 1]
```diff
@@ -171,7 +180,7 @@ - (void)handleKeyUIEventSwizzle:(UIEvent *)event
   UIKeyModifierFlags modifierFlags = 0;
   BOOL isKeyDown = NO;
   long keyCode = 0;
-
+
```
Comment (luacmartins) [inline ios/KeyCommand.mm:183]: Remove space
```suggestion

```

**Review by luacmartins — COMMENTED** [review]
Just one small comment

File: ios/KeyCommand.mm (hunk 1 above)
Comment (azimgd) [reply ios/KeyCommand.mm]: done

**Review by luacmartins — DISMISSED** [review]
LGTM!

**Comment by luacmartins** [comment]
@azimgd it seems like your merge commit is not signed. Could you please sign that commit? You could try the approach below:

```
# (on feature branch) enter interactive rebase
git rebase -i --rebase-merges <commit hash that is unsigned>~

# find your unsigned commit hash, and add a line underneath that says:
exec git commit --amend -S

# save your changes and exit your editor
```

**Comment by azimgd** [comment]
Fixed.

**Comment by luacmartins** [comment]
Just curious, did you use the approach I mentioned above or something else? Asking because another contributor is having a similar issue so that could help them.

**Review by luacmartins — APPROVED** [review]

**Comment by azimgd** [comment]
Approach above, but rebase only signed that particular commit, and unsigned next one. I think I was using this before:
```
// All commits started from the next after <commit_hash> will be rebased with no changes except signing
git rebase --exec 'git commit --amend --no-edit -n -S' -i <commit_hash>
```

**Comment by luacmartins** [comment]
Thanks for confirming!

**Comment by mananjadhav** [comment]
Our order of the conditions caused this [issue](https://github.com/Expensify/App/issues/18480). We should’ve checked for modifiers + enter first and then just enter.

**Pull request merged**